src/parsers/cdm_parser.jl

```diff
@@ -15,14 +15,16 @@ Returns:
     as values
 
 """
-function read_csv_data(file_path::String)
+function read_csv_data(file_path::String, baseMVA::Float64)
     files = readdir(file_path)
     REGEX_DEVICE_TYPE = r"(.*?)\.csv"
     REGEX_IS_FOLDER = r"^[A-Za-z]+$"
     data =Dict{String,Any}()
 
     if length(files) == 0
         @error "No test files in the folder"
```

Comment:
I know this is existing code, but should this error log print be changed to error() (which terminates execution)?